.github/workflows/qns.yml

```
@@ -0,0 +1,46 @@
+name: qns (quic-network-simulator)
+
+on:
+  push:
+    branches: ["main", "docker"] # TODO: update
```

```suggestion
    branches: ["main"]
```

To be removed before merge.